packages/react-dom/src/events/SyntheticKeyboardEvent.js

```diff
@@ -16,6 +16,7 @@ import getEventModifierState from './getEventModifierState';
  */
 const SyntheticKeyboardEvent = SyntheticUIEvent.extend({
   key: getEventKey,
+  code: null,
```

Comment:
In `__DEV__`, could this be made into a getter that would throw if `code` is missing from the event?

Reply:
(1) Yeah it could if the code is missing or incorrect.  
(2) If it did proceed to load it would load wrong and kick the user out of the server and or domain. 
(3) Codes for all links and sites haft to be correctly placed in the script and diagram/program.

Reply:
I added a getter for this to throw an error if `code` is not defined on the `KeyboardEvent` object. I also tried to add tests for this but I couldn't find a way to force code to be undefined on the native event within node, any suggestions/help with that?